bridge_common.spec

```diff
@@ -0,0 +1,67 @@
+%define pkg_name tendrl-bridge-common
+%define pkg_version 0.0.1
+%define pkg_release 1
+
+Name: %{pkg_name}
+Version: %{pkg_version}
+Release: %{pkg_release}%{?dist}
+BuildArch: noarch
+Summary: Common Module for All Bridges
+Source0: %{pkg_name}-%{pkg_version}.tar.gz
+Group:   Applications/System
+License: LGPL2.1
+Url: https://github.com/Tendrl/bridge_common
+
+Requires: python-etcd
+Requires: python-dateutil
+Requires: python-hacking
+Requires: pytest
+Requires: python-pytest-cov
+Requires: python-mock
+Requires: python-sphinx
+Requires: python-reno
+Requires: python-coveralls
+
+%description
+Common python module usable by all Tendrl SDS Bridges
+
+%prep
+%setup -n %{pkg_name}-%{pkg_version}
+# Remove the requirements file to avoid adding into
+# distutils requiers_dist config
+rm -rf {test-,}requirements.txt
+
+# Remove bundled egg-info
+rm -rf %{pkg_name}.egg-info
+
+%build
+%{__python} setup.py build
+
+# generate html docs
+%if 0%{?rhel}==7
+sphinx-1.0-build doc/source html
+%else
+sphinx-build doc/source html
+%endif
+# remove the sphinx-build leftovers
+rm -rf html/.{doctrees,buildinfo}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+%{__python} setup.py install --single-version-externally-managed -O1 --root=$RPM_BUILD_ROOT --record=INSTALLED_FILES
+install -Dm 755 etc/tendrl/tendrl.conf.sample $RPM_BUILD_ROOT/usr/share/tendrl/commons/tendrl.conf
+install -Dm 755 etc/tendrl/tendrl.conf.sample $RPM_BUILD_ROOT/etc/tendrl.conf.sample
+
+%post
+mkdir /var/log/tendrl >/dev/null 2>&1 || :
```

Comment:
Please don't make this directory in `%post`. It should be a `%dir` entry in `%files` instead.